app/scripts/lib/transaction/hooks/enforce-simulation/utils/delegation.ts

```diff
@@ -42,8 +47,8 @@ function nativeTokenMaxLossBuilder(
 
 function erc20MaxLossBuilder(
   _: DeleGatorEnvironment,
-  recipient: Address,
   tokenAddress: Address,
+  recipient: Address,
```

Comment:
Why is this swapped as we're still calling below with the account address first?

Reply:
I simply followed the params order from the enforcers. I asked myself the same question when I saw that `erc20BalanceGte` had the params in different order than `erc721BalanceGte`.

Reply:
Not sure I follow, the terms order hasn't changed as we're encoding below with the recipient first.

But we're changing the caveat builder order here to expect the token address first, but on line `141` below, we're still passing the account address first?

Reply:
Oh, you're right! I thought you were commenting about how erc20 and erc721 caveat builders had different params order. My bad! Should be fixed now! Good catch!